msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -427,6 +427,79 @@ else { /* handle case where capability isn't supported */ }
 
 ```
 
+## Generate a deep link to share content to stage in meetings
+
+You can also generate a deep link to [share the app to stage](~/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md#share-entire-app-to-stage) and start or join a meeting.
+
+> [!NOTE]
+>
+> * Currently, generate a deep link to share content to stage in meetings is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
```

Comment:
Consider updating to: 
Currently, generate a deep link to share content to stage in meetings is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md). **We are working on UX enhancements before making the feature generally available.**

Reply:
Small additional comment after talking internally: let's say:
Currently, the deep link to share content to the stage in meetings is available only in [public developer preview](https://github.com/MicrosoftDocs/msteams-docs/pull/~/resources/dev-preview/developer-preview-intro.md). We are working on UX improvements which will be available before the end of the year.

Reply:
Added the note